gson/src/test/java/com/google/gson/GsonTest.java

```diff
@@ -282,13 +278,8 @@ public void testNewJsonWriter_Default() throws IOException {
     jsonWriter.value(true);
     jsonWriter.endObject();
 
-    try {
-      // Additional top-level value
-      jsonWriter.value(1);
-      fail();
-    } catch (IllegalStateException expected) {
-      assertThat(expected).hasMessageThat().isEqualTo("JSON must have only one top-level value.");
-    }
+    IllegalStateException e = assertThrows(IllegalStateException.class, () -> jsonWriter.value(1));
```

Comment:
Maybe re-add the previous comment:
```suggestion
    // Additional top-level value
    IllegalStateException e = assertThrows(IllegalStateException.class, () -> jsonWriter.value(1));
```